auslib/test/admin/views/test_releases.py

```diff
@@ -1239,6 +1241,35 @@ def testUpdateScheduledChangeExistingRelease(self):
         self.assertEquals(dict(cond[0]), cond_expected)
 
     @mock.patch("time.time", mock.MagicMock(return_value=300))
+    def testUpdateScheduledChangeExistingReleaseResetSignOffs(self):
+        data = {
+            "data": '{"name": "ab", "hashFunction": "sha512", "extv": "3.0", "schema_version": 1}', "name": "ab",
```

Comment:
While adding the "data" column to a scheduled deletion works, it's not really something that's done in real usage. If you're going to continue using sc_id 4 for this test, I'd suggest updating "when" instead. Or you can use a different sc_id for it - up to you!